### PR TITLE
chore: deployment manifest, kaniko

### PR DESCRIPTION
**Pull request by thesephirot**
for seperate base and overlay config we move the base to software repo

### CODE REVIEW

File: k8s/base/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    metadata:
+      labels:
+        app: zitadel
+        version: v1
+    spec:
+      containers:
+      - name: zitadel
+        image: docker.pkg.github.com/caos/zitadel/zitadel:latest
+        imagePullPolicy: Always
+        args: ["-login=false", "-console=false"]
```
Comment (fforootd) [inline k8s/base/deployment.yaml:23]: we should no supply this as standard args

File: k8s/base/flow.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            reserve_data: true
+            parse:
+                type: logfmt
+      - tag_normaliser:
+            metadata: ${namespace}.${container}.${pod}
+            metadata_name: ${namespace_name}.${container_name}.${pod_name}
+    selectors:
+        app: zitadel
+    outputRefs:
+      - output-loki
```
Comment (fforootd) [inline k8s/base/flow.yaml:18]: newline break

File: k8s/base/output.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (fforootd) [inline k8s/base/output.yaml:1]: empty line

File: k8s/base/service-api-mangagement.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  ports:
+  - name: rest
+    port: 80
+    targetPort: management-rest
+  - name: grpc
+    port: 8080
+    targetPort: management-grpc
+  selector:
+    app: zitadel
+    version: v1
```
Comment (fforootd) [inline k8s/base/service-api-mangagement.yaml:16]: newline

File: k8s/base/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  template:
+    metadata:
+      labels:
+        app: zitadel
+        version: v1
+    spec:
+      containers:
+      - name: zitadel
+        image: docker.pkg.github.com/caos/zitadel/zitadel:latest
+        imagePullPolicy: Always
```
Comment (fforootd) [inline k8s/base/deployment.yaml:22]: i dont think always is good, we should use `ifNotPresent`

Comment (thesephirot) [reply k8s/base/deployment.yaml:22]: that is fine with me

File: k8s/base/deployment.yaml [hunk 6]
```diff
@@ -19,7 +19,7 @@ spec:
       containers:
       - name: zitadel
         image: docker.pkg.github.com/caos/zitadel/zitadel:latest
-        imagePullPolicy: Always
+        imagePullPolicy: ifNotPresent
```
Comment (fforootd) [inline k8s/base/deployment.yaml:22]:
```suggestion
        imagePullPolicy: IfNotPresent
```

Comment (thesephirot) [reply k8s/base/deployment.yaml:22]: copy pasta :)

File: k8s/base/deployment.yaml (hunk 1 above)
Comment (thesephirot) [reply k8s/base/deployment.yaml:23]: master does currently not proceed without these flags. once master is solid we will remove it anyway.

Comment (fforootd) [reply k8s/base/deployment.yaml:23]: in which PR will this happen @livio-a @fgerschwiler? because we should reference it for further tracking

**Comment by fforootd** [comment]
@thesephirot please use `semrel` compatible titles for PR, i did change it already

**Pull request merged**